src/Microsoft.DotNet.Arcade.Sdk/src/InstallDotNetCore.cs

```diff
@@ -118,6 +122,16 @@ public override bool Execute()
                                             arguments += $" -architecture {architecture}";
                                         }
 
+                                        if (!String.IsNullOrWhiteSpace(NetRuntimeSourceFeed))
```

Comment:
\<nit> The style in this file seems to be to use the alias. (That's generally the style I see elsewhere, too.)

```suggestion
                                        if (!string.IsNullOrWhiteSpace(NetRuntimeSourceFeed))
```

Same for the next one.